WordPress/Classes/ViewRelated/Post/PostPreviewViewController.m

```diff
@@ -180,7 +180,7 @@ - (void)webView:(UIWebView *)webView didFailLoadWithError:(NSError *)error
 
     [self stopLoading];
 
-    [self.generator previewRequestFailedWithError:error];
+    [self.generator previewRequestFailedWithMessage:error.localizedDescription];
```

Comment:
I'm concerned about sending this error as-is, and potentially leaking private information. Also because it might get too noisy if it's also generated by normal connectivity issues. If you think there's still value on getting those, I'd log only the error domain and code.